src/Components/test/E2ETest/Tests/AuthTest.cs

```diff
@@ -188,6 +191,11 @@ public void Router_RequireRole_NotAuthorized()
         protected IWebElement MountAndNavigateToAuthTest(string authLinkText)
         {
             Navigate(ServerPathBase);
+
+            Browser.Manage().Cookies.DeleteCookieNamed("WebSockets.Identifier");
+            Browser.Manage().Cookies.AddCookie(new Cookie("WebSockets.Identifier", SessionIdentifier));
+            Browser.Navigate().Refresh();
+
```

Comment:
Is there a drawback to this running for the client-side tests?

Reply:
These test don't run on the client, when they do, we can simply key this off to whether or not we are running server-side.